### PR TITLE
Misc fix for seesaw

### PR DESCRIPTION
**Pull request by liuyuan10**
Three misc fix for seesaw

### CODE REVIEW

File: binaries/seesaw_cli/main.go [hunk 1]
```diff
@@ -184,9 +188,11 @@ func interactive() {
 		if cmdline == "" {
 			continue
 		}
+		terminal.Restore(syscall.Stdin, oldTermState)
```
Comment (anfernee) [inline binaries/seesaw_cli/main.go]: what's the symptom if you don't do this? Especially it's only changing `syscall.Stdin`. `fmt.Print` prints to `stdout` by default.

Comment (liuyuan10) [reply binaries/seesaw_cli/main.go]: The symptom is fmt.Println doesn't go back to the begining of a line. terminal.Restore applies to whole terminal not just stdin.

Comment (anfernee) [reply binaries/seesaw_cli/main.go]: interesting to know that..

**Pull request merged**